build_all/base-libraries/AzureIoTUtility/src/esp32/sample_init.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifdef ARDUINO_ARCH_ESP32
+
+#include <Arduino.h>
+#include <time.h>
+#include "AzureIoTSocket_WiFi.h"
+#include <WiFi.h>
+#include <WiFiClientSecure.h>
+#include <WiFiUdp.h>
+
+// Times before 2010 (1970 + 40 years) are invalid
+#define MIN_EPOCH 40 * 365 * 24 * 3600
```

Comment:
>40 * 365 * 24 * 3600 [](start = 18, length = 20)

please add curly brace around macro value.

Reply:
It works, but why?

Reply:
Generally a good practice to wrap multi member macro’s in parenthesis to make explicit its intention. [Here’s a good example where not doing it could cause unintended behavior](https://stackoverflow.com/questions/10820340/the-need-for-parentheses-in-macros-in-c). It may work here but there could be future macros that aren’t so lucky.

Reply:
Wait @hihigupt curly brace or parenthesis?

Reply:
For good practice
Check out this 
[https://stackoverflow.com/questions/321143/good-programming-practices-for-macro-definitions-define-in-c](https://stackoverflow.com/questions/321143/good-programming-practices-for-macro-definitions-define-in-c)

---
In reply to: [350980877](https://github.com/Azure/azure-iot-pal-arduino/pull/7#discussion_r350980877) [](ancestors = 350980877)

Reply:
parenthesis...:)

---
In reply to: [351018033](https://github.com/Azure/azure-iot-pal-arduino/pull/7#discussion_r351018033) [](ancestors = 351018033)

Reply:
Fixed.